src/Workspaces/Core/Portable/Remote/RemoteFeatureOptions.cs

```diff
@@ -11,49 +10,10 @@ internal static class RemoteFeatureOptions
     {
         private const string LocalRegistryPath = @"Roslyn\Features\Remote\";
 
-        /// <summary>
-        /// Global switch that determines of OOP can be used for a language feature. Exposed through
-        /// user visible switch to let people opt-in/out of this behavior.
-        /// </summary>
-        public static readonly Option<bool> OutOfProcessAllowed = new Option<bool>(
-            nameof(RemoteFeatureOptions), nameof(OutOfProcessAllowed), defaultValue: false,
-            storageLocations: new LocalUserProfileStorageLocation(LocalRegistryPath + nameof(OutOfProcessAllowed)));
-
-        // Individual feature switches.  Not exposed to the user.  Supplied as an escape hatch for
-        // features if necessary.  If all features use OOP then no indices will need to be built
-        // within VS.  However, if any features need to run in VS, then we have to build our indices
-        // in VS as well.
-
-        public static readonly Option<bool> AddImportEnabled = new Option<bool>(
-            nameof(RemoteFeatureOptions), nameof(AddImportEnabled), defaultValue: true,
-            storageLocations: new LocalUserProfileStorageLocation(LocalRegistryPath + nameof(AddImportEnabled)));
-
-        public static readonly Option<bool> DocumentHighlightingEnabled = new Option<bool>(
-            nameof(RemoteFeatureOptions), nameof(DocumentHighlightingEnabled), defaultValue: true,
-            storageLocations: new LocalUserProfileStorageLocation(LocalRegistryPath + nameof(DocumentHighlightingEnabled)));
-
-        public static readonly Option<bool> NavigateToEnabled = new Option<bool>(
-            nameof(RemoteFeatureOptions), nameof(NavigateToEnabled), defaultValue: true,
-            storageLocations: new LocalUserProfileStorageLocation(LocalRegistryPath + nameof(NavigateToEnabled)));
-
-        public static readonly Option<bool> SymbolSearchEnabled = new Option<bool>(
-            nameof(RemoteFeatureOptions), nameof(SymbolSearchEnabled), defaultValue: true,
-            storageLocations: new LocalUserProfileStorageLocation(LocalRegistryPath + nameof(SymbolSearchEnabled)));
-
-        public static readonly Option<bool> SymbolFinderEnabled = new Option<bool>(
-            nameof(RemoteFeatureOptions), nameof(SymbolFinderEnabled), defaultValue: true,
-            storageLocations: new LocalUserProfileStorageLocation(LocalRegistryPath + nameof(SymbolFinderEnabled)));
-
         public static readonly Option<bool> DiagnosticsEnabled = new Option<bool>(
             nameof(RemoteFeatureOptions), nameof(DiagnosticsEnabled), defaultValue: true,
             storageLocations: new LocalUserProfileStorageLocation(LocalRegistryPath + nameof(DiagnosticsEnabled)));
```

Comment:
this option is still around since some tests explicitly set this to false as diagnostics doesn't work OOP in tests (not sure why, but i'm just preserving that behavior).

Reply:
@heejaechang Do you know why diagnostics can't run OOP?  The relevant check that forces us to keep this is here:
https://github.com/dotnet/roslyn/blob/f79608d488bcb63336be0d50ced3a87c77108e4c/src/EditorFeatures/TestUtilities/CodeActions/AbstractCodeActionOrUserDiagnosticTest.cs#L375-L379

Reply:
test doesn't have OOP. you need to explicitly set up mock remote host to simulate OOP in unit test which I do for VS.Net unit test but nowhere else.